test_scripts/API/AddCommand/005_ATF_AddCommand_UI_AddCommand_WARNINGS.lua

```diff
@@ -0,0 +1,142 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this particular test it is checked case when SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to UI.AddCommand and VR.AddCommand gets ANY successfull result code
+-- 1. Used preconditions: App is registered and activated SUCCESSFULLY
+-- 2. Performed steps: 
+-- MOB -> SDL: sends AddCommand
+-- SDL -> HMI: resends VR.AddCommand and UI.AddCommand
+-- HMI -> SDL: UI.AddCommand (SUCCESS), VR.AddCommand (cyclically checked cases fo result codes SUCCESS, WARNINGS, WRONG_LANGUAGE, RETRY, SAVED)
+--
+-- Expected result:
+-- SDL -> MOB: AddCommand (result code: WARNINGS, success: true)
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.SDLStoragePath = config.pathToSDL .. "storage/"
```

Comment:
@vvvakulenko, please update config.pathToSDL with commonPreconditions:GetPathToSDL().
Important: You should use this in all scripts, because CI will not find path config.pathToSDL and run-time error/fake fails will be used.

After update please move this line after reference "local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')"